yaksh/templates/yaksh/post_comments.html

```diff
@@ -0,0 +1,72 @@
+{% extends "user.html" %}
+{% load static %}
+{% load humanize %}
```

Comment:
Just a suggestion, You can use `timesince` template tag instead of humanize. For reference, check this https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#timesince